polyfill/test/validStrings.mjs

```diff
@@ -449,31 +448,33 @@ const comparisonItems = {
   YearMonth: ['year', 'month', 'calendar'],
   ZonedDateTime: [...dateItems, ...timeItems, 'offset', 'ianaName', 'calendar']
 };
-const plainModes = ['Date', 'DateTime', 'MonthDay', 'Time', 'YearMonth'];
+const plainModes = ['Instant', 'Date', 'DateTime', 'MonthDay', 'Time', 'YearMonth'];
```

Comment:
This leaves TimeZone and ZonedDateTime untested. Also, see the previous comment because if you add Instant, these are no longer "Plain modes". I'd recommend leaving `plainModes` as it is, and instead looping through `Object.keys(goals)` to get the full list of modes.

Reply:
I overlooked `goals`. I've done a bit of refactoring and restored the original logic.